ios/Classes/AblyStreamsChannel.m

```diff
@@ -99,7 +99,7 @@ - (void)setStreamHandlerFactory:(NSObject<FlutterStreamHandler> *(^)(id))factory
 }
 
 - (void) reset{
-    for (NSString* key in _streams) {
+    for (NSString* key in [_streams mutableCopy]) {
```

Comment:
I know this has been merged but, fwiw, that could just be `copy` as it's not needed to be mutable. Cosmetic, admittedly. 😁

Reply:
@QuintinWillison Good to know there is an immutable copy for dicts 😄 

Addressed at efc5104 in PR #20